dynamiq/nodes/node.py

```diff
@@ -256,9 +257,15 @@ def _validate_dependency_status(depend: NodeDependency, depends_result: dict[str
             )
 
         if dep_result.status == RunnableStatus.SKIP:
-            raise NodeSkippedException(
-                failed_depend=depend, message=f"Dependency {depend.node.id}: skipped"
-            )
+            raise NodeSkippedException(failed_depend=depend, message=f"Dependency {depend.node.id}: skipped")
+
+    @property
+    def input_schema(self) -> BaseModel | None:
+        return self._input_schema
+
+    @classmethod
+    def get_input_schema(cls) -> BaseModel | None:
```

Comment:
It's redundant. `Node.input_schema` and `node_instance.input_schema` should be enough